src/o365/yammer/commands/report/report-groupactivityfilecounts.spec.ts

```diff
@@ -0,0 +1,95 @@
+import commands from '../../commands';
+import Command from '../../../../Command';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+const command: Command = require('./report-groupactivityfilecounts');
+import * as assert from 'assert';
+import Utils from '../../../../Utils';
+import request from '../../../../request';
+
+describe(commands.YAMMER_REPORT_GROUPACTIVITYCOUNTS, () => {
```

Comment:
Renamed from `YAMMER_REPORT_GROUPACTIVITYCOUNTS` to `YAMMER_REPORT_GROUPSACTIVITYCOUNTS` to match the issue description.